src/components/Card/card.css

```diff
@@ -0,0 +1,49 @@
+
+@media screen and (max-width: 600px) {
+  /* Mobile View */
+  .card {
+    max-width: 80%;
+    min-width: 80%;
+    height: 400px;
+    background-color: #355878;
```

Comment:
Should we keep the backend of the card grey so it looks cleaner? and use code sydney lightblue when we hover or click on the card?